include/crm/cib/util.h

```diff
@@ -23,39 +23,10 @@ xmlNode *createEmptyCib(int cib_epoch);
 
 gboolean cib_version_details(xmlNode * cib, int *admin_epoch, int *epoch, int *updates);
 
-int update_attr_delegate(cib_t * the_cib, int call_options,
-                         const char *section, const char *node_uuid,
-                         const char *set_type, const char *set_name,
-                         const char *attr_id, const char *attr_name,
-                         const char *attr_value, gboolean to_console,
-                         const char *user_name, const char *node_type);
-
-int find_nvpair_attr_delegate(cib_t * the_cib, const char *attr,
-                              const char *section, const char *node_uuid,
-                              const char *set_type, const char *set_name,
-                              const char *attr_id, const char *attr_name,
-                              gboolean to_console, char **value, const char *user_name);
-
-int read_attr_delegate(cib_t * the_cib,
-                       const char *section, const char *node_uuid,
-                       const char *set_type, const char *set_name,
-                       const char *attr_id, const char *attr_name,
-                       char **attr_value, gboolean to_console, const char *user_name);
-
-int delete_attr_delegate(cib_t * the_cib, int options,
-                         const char *section, const char *node_uuid,
-                         const char *set_type, const char *set_name,
-                         const char *attr_id, const char *attr_name,
-                         const char *attr_value, gboolean to_console, const char *user_name);
-
```

Comment:
I just realized we don't have any public API replacement for these yet. We should wait to deprecate them until we move cibadmin's functionality to libpacemaker. Sorry for jumping the gun ...

Reply:
Dropped.